linux/installation/mde_installer.sh

```diff
@@ -44,10 +44,10 @@ script_exit()
 detect_distro()
 {
     if [ -f /etc/os-release ]; then
-        if [[ $(grep -o -i "amazon_linux:2" | /etc/os-release) ]]; then
+        if [[ $(grep -o -i "amazon_linux:2" /etc/os-release) ]]; then
             DISTRO='rhel'
             VERSION=7
-        elif
+        else
             . /etc/os-release
```

Comment:
i didnt see your PR before submitting one myself, but this line also needs to be removed as we have already checked if it exists and in this branch of the else we know it doesnt exist so it cant be sourced.